lib/remote/apilistener-filesync.cpp

```diff
@@ -468,7 +466,7 @@ Value ApiListener::ConfigUpdateHandler(const MessageOrigin::Ptr& origin, const D
 		}
 
 		Log(LogInformation, "ApiListener")
-			<< "Applying configuration file update for path '" << stageConfigZoneDir << "' ("
+			<< "AppStage: Updating received configuradate for path '" << stageConfigZoneDir << "' ("
```

Comment:
Why the new term `AppStage`? Please leave the log message "as is", this follows known patterns from 2.10.x.

Reply:
I don't know where that comes, probably a rebase fail

Reply:
Has been changed back